R/pipe_continuation_linter.R

```diff
@@ -0,0 +1,41 @@
+#' @describeIn linters check that all commas are followed by spaces, but do not
+#' have spaces before them.
+#' @importFrom xml2 xml_find_all as_list
+#' @export
+pipe_continuation_linter <- function(source_file) {
```

Comment:
We do include specials in the `infix_spaces_linter`, and this seems to be working fine with pipes, so I think the space aspect is covered.